lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
```

Comment:
Couldn't get `import { values } from 'underscore-plus'` to work for me. Pretty much have no idea when it comes to ES6 modules and require. Need my hand held. We don't need scopes there btw.